Resources/config/controllers.xml

```diff
@@ -28,7 +28,7 @@
             <argument type="service" id="sulu.content.mapper"/>
             <argument type="service" id="sulu_headless.structure_resolver"/>
             <argument type="service" id="jms_serializer.serializer"/>
-            <argument type="service" id="sulu_headless.reference_store.snippet_area"/>
+            <argument type="service" id="sulu_snippet.reference_store.snippet_area"/>
```

Comment:
for compatibility to lower sulu versions I think we need `on-invalid="null"`

Reply:
fixed